src/main/java/com/hubspot/jinjava/el/ext/RegularDivOperator.java

```diff
@@ -0,0 +1,39 @@
+package com.hubspot.jinjava.el.ext;
+
+import de.odysseus.el.misc.TypeConverter;
+import de.odysseus.el.tree.impl.Parser.ExtensionHandler;
+import de.odysseus.el.tree.impl.Parser.ExtensionPoint;
+import de.odysseus.el.tree.impl.Scanner;
+import de.odysseus.el.tree.impl.ast.AstBinary;
+import de.odysseus.el.tree.impl.ast.AstBinary.SimpleOperator;
+import de.odysseus.el.tree.impl.ast.AstNode;
+
+public class RegularDivOperator extends SimpleOperator {
```

Comment:
I think this could simply be called `DivOperator`.

Also I think it would be worth a comment on why this was overridden.